docs/components/compact.md

```diff
@@ -60,7 +60,7 @@ Flags:
       --retention.resolution-1h=0d  
                            How long to retain samples of resolution 2 (1 hour)
                            in bucket. 0d - disables this retention
-  -w, --wait               Do not exit after all compactions have been processed
-                           and wait for new work.
+  -r, --repeat=5m          Do not exit after all compactions have been processed
```

Comment:
Can we move the name in different PR and discuss about it? Sure we are in minor release but we cannot break compatiblity that easily in every PR. Someday there will be another contributor who will change it in random PR to something else. This one makes sense, just we need to be nice to users and not break it. Plus changelog change is must have here.

Reply:
>This one makes sense, just we need to be nice to users and not break it.
So you suggest to leave `--wait` with a default 5m interval here?

I understand you concern, but splitting this changes it 2 requests looks not to be convenient. PR has exactly the purpose to discuss changes and find tradeoff 😀

Reply:
yea.. maybe you are right just I was overwhelmed how many things you are changing here just to fix retries. So .. let's discuss it here (: 

When you think about `repeat` flag. I think we don't need that one. Why you even care how often it is running? IMO there is no reason to control this. Compactor should basically run all the time, maybe even as often as needed. So I would not change this. What do you think? (:

Reply:
anyway, even if you don't need customise compactor (though for the small setup no need to scan blocks' meta every 5 minutes, you can do it once per hours), the `wait` flag seems not to be appropriate in conjunction with time intervals passed to it